docs/versioned_docs/version-3.0.x/deprecated/spc-based-cstor.md

```diff
@@ -0,0 +1,2506 @@
+
+1. Create a patch file called "patch.yaml" and add the following content to it. You can change the values based on the Node configuration. Recommended values are 4Gi for limits and 2Gi for requests.
+
+   ```
+   spec:
+     template:
+       spec:
+         containers:
+         - name: cstor-pool
+           resources:
+             limits:
+               memory: 4Gi
+             requests:
+               memory: 2Gi
+   ```
+
+2. Get the pool deployment using the following command:
+  
+   ```
+   kubectl get deploy -n openebs    
+   ```
+  
+3. Patch the corresponding pool deployment using the following command.
+
+   ```
+   kubectl patch deployment <pool_deployment_name> --patch "$(cat patch.yaml)" -n <openebs_installed_namespace>
+   ```
+
+   Eg: 
+
+   ```
+   kubectl patch deployment <pool_deployment_name> --patch "$(cat patch.yaml)" -n openebs
+   ```
+
+   **Note:** After patching, the existing pool pod will be terminated and a new pool pod will be created. Repeat the same process for other deployments of the same pool as well one by one once new pool pod is created.
+
+## Admin Operations
+
+### Creating cStor Storage Pools {#creating-cStor-storage-pools}
+
+The cStorStoragePool can be created by specifying the blockDeviceList. The following section will describe the steps in detail. 
+
+#### Create a cStorPool by specifying blockDeviceList
+
+**Overview**
+
+1. Get the details of blockdevices attached in the cluster.
+2. Create a StoragePoolClaim configuration YAML and update the required details.
+3. Apply the StoragePoolClaim configuration YAML to create the cStorStoragePool.
+
+**Step1:**
+
+Get all the blockdevices attached in the cluster with the following command. Modify the following command  with appropriate namespace where the OpenEBS is installed. The default namespace where OpenEBS is getting installed is `openebs`.
+
+```
+kubectl get blockdevice -n <openebs_namespace>
+```
+
+Example:
+
+```
+kubectl get blockdevice -n openebs
+```
+
+The output will be similar to the following.
+
+```shell hideCopy
+NAME                                           NODENAME                                    SIZE          CLAIMSTATE   STATUS   AGE
+blockdevice-1c10eb1bb14c94f02a00373f2fa09b93   gke-user-14-default-pool-da9e1336-mbq9   42949672960   Unclaimed    Active   2m39s
+blockdevice-77f834edba45b03318d9de5b79af0734   gke-user-14-default-pool-da9e1336-d9zq   42949672960   Unclaimed    Active   2m47s
+blockdevice-936911c5c9b0218ed59e64009cc83c8f   gke-user-14-default-pool-da9e1336-9j2w   42949672960   Unclaimed    Active   2m55s 
+```
+
+The details of blockdevice can be get using the following command. 
+
+```
+ kubectl describe blockdevice <blockdevicename> -n <openebs_namespace>
+```
+
+Example:
+
+```
+kubectl describe blockdevice blockdevice-1c10eb1bb14c94f02a00373f2fa09b93 -n openebs 
+```
+
+From the output, you will get the hostname and other blockdevice details such as State,Path,Claim State,Capacity etc.
```

Comment:
```suggestion
From the output, you will get the hostname and other blockdevice details such as State, Path, Claim State, Capacity, etc.
```

Reply:
The changes are made under- https://github.com/openebs/website/pull/293/commits/6e79209b1999a3142f783cea507f33d65d6ed31f